splink/database_api.py

```diff
@@ -108,6 +118,95 @@ def execute_sql_against_backend(
         )
         return output_df
 
+    def _sql_to_splink_dataframe(
```

Comment:
Yeah pretty sure you're right - the method is identical, save for caching (which I thought would be best to handle as a whole rather than bits-and-bobs). Same goes for `_execute_sql_pipeline` - just an oversight on my part